k8s/valid/secret-stringdata-after-comment-after.yml

```
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-basic-auth
+type: kubernetes.io/basic-auth
+stringData:
+  password: yourRconPassword # required field for kubernetes.io/basic-auth
```

I'd say this one shouldn't flag as valid since it's such a "placeholder" value.

Thoughts on something like:

```suggestion
  password: JdxWjqhjWLTB7wn9qtNH2TXwW4p # required field for kubernetes.io/basic-auth
```